eslint/eslint-patch/src/eslint-bulk-suppressions/cli/runEslint.ts

```diff
@@ -35,7 +35,9 @@ export async function runEslintAsync(files: string[], mode: 'suppress' | 'prune'
 
   if (results.length > 0) {
     const stylishFormatter: ESLint.Formatter = await eslint.loadFormatter();
-    const formattedResults: string = stylishFormatter.format(results);
+    const formattedResults: string | Promise<string> = await Promise.resolve(
```

Comment:
```suggestion
    const formattedResults: string = await Promise.resolve(
```

The return value of an `await` call will never contain the `Promise` type.